### ISSUE TITLE
Demo broken

### ISSUE THREAD

**Issue by jvolker** [davestewart/nuxt-content-assets#44]
The demo seems broken. 

While starting the app it throws: `Cannot start nuxt:  Cannot find module '@nuxt/image-edge' `
It happens following the instructions of the Readme installing locally

```
git clone https://github.com/davestewart/nuxt-content-assets.git
cd nuxt-content-assets
npm install
npm run dev
```

as well as using the provided link: https://stackblitz.com/github/davestewart/nuxt-content-assets?file=demo%2Fapp.vue

**Comment by davestewart** [comment]
Sorry folks, have been away from oss for a bit. Let me check and get back to you

**Comment by davestewart** [comment]
OK, on this today. Will release shortly

**Comment by jvolker** [comment]
Thanks for fixing this. Following the instructions, the demo seems to work for the most part now. Only the Nuxt Image component example throws this error: `Uncaught (in promise) TypeError: $img is not a function`

Also, the online demo is not working, throwing this error:
`[23:54:41]  ERROR  Cannot start nuxt:  Cannot find module '/home/projects/wmaikrxvl.github/demo/@nuxt/image'`

**Comment by davestewart** [comment]
Hi Jeremias,

I can't reproduce the Nuxt image bug here.

Could you raise a new ticket (be sure to use the `bug` template) and provide full details, including:

- error stack
- where the bug appears (client, server, cli, etc)
- when the bug appears (build, navigating to page, etc)

I'll check the demo as well.

Thanks!

**Comment by davestewart** [comment]
OK, looking at the demo, the issue is certainly with the Nuxt Image import.

This was always a bit of a hack I guess I need to find a better way to load the `NuxtImg` component on demand.

Ideas welcome!

**Comment by davestewart** [comment]
I've put the above code into a branch:

- [bug/nuxt-img](https://github.com/davestewart/nuxt-content-assets/tree/bug/nuxt-img)

See `<nuxt-img />` here:

- [bug/nuxt-img/demo/content/main.md](https://raw.githubusercontent.com/davestewart/nuxt-content-assets/bug/nuxt-img/demo/content/main.md)

```md
---
title: Nuxt Content Assets - Demo
---

<nuxt-img src="splash.png" style="margin: -20px 0" />

# Demo

...
```

**Comment by ThomasWT** [comment]
Caught you on Twitter but leaving the solution here. 

Should be solved by running `npm install` in both the root directory and in ./demo

https://stackblitz.com/edit/github-wc2y8l-tfy7ap?file=.stackblitzrc

**Comment by davestewart** [comment]
Closed by #54

**Comment by jvolker** [comment]
Thanks for fixing this. 

> Should be solved by running npm install in both the root directory and in ./demo

Would be good to add this to the [setup instructions](https://github.com/davestewart/nuxt-content-assets#demo).
